server/service/src/processors/transfer/invoice/mod.rs

```diff
@@ -93,6 +94,60 @@ pub(crate) enum ProcessInvoiceTransfersError {
     NameIsNotAnActiveStore(ChangelogRow),
 }
 
+fn log_system_error(
+    connection: &StorageConnection,
+    error: &ProcessInvoiceTransfersError,
+) -> Result<(), ProcessInvoiceTransfersError> {
+    let error_message = format!("ProcessInvoiceTransfersError: {:?}", error);
```

Comment:
Not 100% sure how this would look as database change log, but [format_error](https://github.com/msupply-foundation/open-msupply/blob/2fd3228f7bd8cc40acbd16e71c9a987c31c2d9b8/server/util/src/error.rs#L36) is usually better for displaying to user